src/main/resources/application.yml

```diff
@@ -3,7 +3,12 @@ server:
 
 resttemplate:
   httpclient:
-    timeout:10000
+    timeout: 10000
+
+ftp.client:
+  url: https://green.blob.core.windows.net
```

Comment:
does this need to be in vault too?

Reply:
apparently not, i asked them what we would need to keep private, no one can do anything as long as they dont have the signature, 
however if you feel i should put them there then i can do that too!

Reply:
Nope!